jcl/src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -235,4 +235,18 @@ public static void reachabilityFence(java.lang.Object ref) {
 }
 /*[ENDIF]*/
 
+/*[IF Java11]*/
+/**
+ * This method will always throw CloneNotSupportedException. A clone of this instance will not be returned 
+ * since a Reference cannot be cloned. Workaround is to create a new Reference.
+ * 
+ * @throws CloneNotSupportedException
+ *
+ * @since 11
+ */
+protected Object clone() throws CloneNotSupportedException {
```

Comment:
This causes a warning, please tag with `@Override`

Reply:
added `@Override`